src/analytics/Cloud.js

```diff
@@ -7,9 +7,9 @@ const Cloud = ({ cards }) => {
   cards.forEach((card) =>
     card.tags.forEach((tag) => {
       if (tags[tag]) {
-        tags[tag] += card.asfan;
+        tags[tag] += 1;
```

Comment:
We can address this in a future PR, but I'd very much like this to be able to be based off asfan or count as an option to the user. I can take that up though with some other improvements I wanted to do to it.